straxen/plugins/peaks/peak_positions_cnf.py

```diff
@@ -210,10 +210,19 @@ def compute(self, peaks):
         r_max = np.max(r_array, axis=1)
 
         theta_array = np.arctan2(contours[..., 1], contours[..., 0])
-        theta_min = np.min(theta_array, axis=1)
-        theta_max = np.max(theta_array, axis=1)
+
+        # Correction for circular nature of angle
+        avg_theta = np.arctan2(
+            np.mean(contours[..., 1], axis=1),
+            np.mean(contours[..., 0], axis=1),
+        )
+
+        avg_theta = np.reshape(avg_theta, (avg_theta.shape[0], 1))
+        theta_array_shift = (np.subtract(theta_array, avg_theta) + np.pi) % (2 * np.pi)
```

Comment:
I guess `theta_array - avg_theta` is better than `np.subtract(theta_array, avg_theta)`.